api/room/schema.py

```diff
@@ -20,6 +21,15 @@ class Meta:
         model = RoomModel
 
 
+def save_room_tags(room, room_tags):
+    for tag in room_tags:
+        room_tag = TagModel.query.filter_by(id=tag).first()
+        if not room_tag:
```

Comment:
Since there are more than one tag coming from the user, would it be appropriate to be more specific in the error message?

Reply:
I have updated the code to state all the tags that are invalid @Bonifase. Thanks for pointing that out.